modules/gdscript/gdscript_editor.cpp

```diff
@@ -3456,6 +3456,30 @@ ::Error GDScriptLanguage::lookup_code(const String &p_code, const String &p_symb
 				return OK;
 			}
 		} break;
+		case GDScriptParser::COMPLETION_TYPE_ATTRIBUTE: {
+			const GDScriptParser::TypeNode *type = static_cast<const GDScriptParser::TypeNode *>(context.node);
```

Comment:
Is `type` guaranteed to be not null in here? If not, it needs to be ensured right after.

Reply:
I added a check to be safe